### ISSUE TITLE
computerDetails.building and computerDetails.department are retrieving ID's instead of names

### ISSUE THREAD

**Issue by guliciuc** [jamf/powerbi#38]
Hi

While trying to move my old dashboard to the new release, i noticed that computerDetails.building and computerDetails.department are retieving the ID's instead of the names. IS anyone  having the same problem or is it just me?

Regards
Adrian

**Comment by jfretheim** [comment]
Yes, you have found an issue here. The new version will display these as IDs rather than names due to data model differences between the Classic and Jamf Pro APIs. So, any existing reports based on those fields will need to account for that.

The new connector would need to make separate API calls to lookup departments and buildings to obtain the names (which would then require additional Read Buildings and Read Departments privileges for the user or API Client configured for the connection).

Definitely something to look into for a future release. Thank you very much for reporting this!

**Comment by guliciuc** [comment]
Hi

I'm just curios, is there a plan to fix this in the near future?

Regards
Adrian

**Comment by jfretheim** [comment]
> Hi
> 
> I'm just curios, is there a plan to fix this in the near future?
> 
> Regards Adrian

No immediate plans to work on this at this time.
